Assets/Scripts/CombatManager.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CombatManager : MonoBehaviour
+{
+    public GameObject playerPrefab;
+    public GameObject enemyPrefab;
+
+    public CastButtonController castButton;
+
+    public int cardsRequired;
+
+    CreatureController target;
+
+    int cardsSelected;
+
+    void Start()
+    {
+        GenerateCombatParticipants();
+    }
+
+    void GenerateCombatParticipants() {
+        // Instantiate prefabs
+        GameObject player = GenerateCreature(playerPrefab);
+        GameObject enemy = GenerateCreature(enemyPrefab);
+    }
+
+    GameObject GenerateCreature(GameObject creaturePrefab) {
+        GameObject creature = Instantiate(creaturePrefab);
+
+        creature.transform.SetParent(gameObject.transform, false);
+
+        creature.GetComponent<CreatureController>().combatManager = this;
+
+        return creature;
+    }
+
+    public void SetTarget(CreatureController target) {
+        this.target = target;
+
+        CheckCastReady();
+    }
```

Comment:
If I understand correctly this will only update the current reference to the CombatManager's CreatureController, `target` to the most recently clicked Creature.  However, we are able to select multiple creatures which I believe we agreed is intended.  Should we instead collect a list of targets to handle these cases instead of just having a single target reference?

Reply:
Agreed. Changed behavior to collect a list of targets.